source/mpi_fsi.cpp

```diff
@@ -488,14 +479,12 @@ namespace MPI
                          "There should be only 2 groups of finite element!"));
                 if (i_group == 1)
                   continue; // skip the pressure dofs
-                bool inside = true;
+                int inside = 0;
```

Comment:
Since now this is a count, let's rename it to something more precise, for example, `inside_dim_count`.